invenio_oauthclient/ext.py

```diff
@@ -127,3 +127,16 @@ def init_config(self, app):
         for k in dir(config):
             if k.startswith('OAUTHCLIENT_'):
                 app.config.setdefault(k, getattr(config, k))
+
+        # Override template configuration.
+        template_key = app.config.get(
+            'OAUTHCLIENT_TEMPLATE_KEY',
+            'SECURITY_LOGIN_USER_TEMPLATE'  # default template key
+        )
+        if template_key is not None:
```

Comment:
@tiborsimko please have a look at the logic here. It's quite tricky if you want to keep the templates configurable.